default.nix

```diff
@@ -54,11 +54,23 @@ let
 
           proto3-wire =
             pkgs.haskell.lib.dontCheck
-              (self.callPackage ./nix/proto3-wire.nix { });
```

Comment:
The proto3-wire version has not changed, but I moved it into `default.nix` for the sake of simplicity. The proto3-suite version was bumped to support newer Swagger and QuickCheck.